packages/gdb.rb

```diff
@@ -6,18 +6,18 @@
 class Gdb < Autotools
   description 'The GNU Debugger'
   homepage 'https://www.gnu.org/software/gdb/'
-  version '14.2-py3.12'
+  version '15.1-py3.12'
   license 'GPL3'
   compatibility 'all'
-  source_url 'https://ftpmirror.gnu.org/gnu/gdb/gdb-14.2.tar.xz'
-  source_sha256 '2d4dd8061d8ded12b6c63f55e45344881e8226105f4d2a9b234040efa5ce7772'
+  source_url 'https://ftpmirror.gnu.org/gnu/gdb/gdb-15.1.tar.xz'
```

Comment:
We can probably use `#{version.split('-').first}` here, right?

Reply:
Done.